app/components/portable-infobox.js

```diff
@@ -42,9 +43,17 @@ export default Component.extend(
 		collapsedHeight: computed('viewportDimensions.width', 'viewportDimensions.height', function () {
 			const deviceWidth = this.get('viewportDimensions.width'),
 				deviceHeight = this.get('viewportDimensions.height'),
-				isLandscape = deviceWidth > deviceHeight;
+				isLandscape = deviceWidth > deviceHeight,
+				calculatedHeight = Math.floor((isLandscape ? deviceHeight : deviceWidth) * 16 / 9) + 100;
 
-			return Math.floor((isLandscape ? deviceHeight : deviceWidth) * 16 / 9) + 100;
+			/**
+			 * FIXME FEATURED VIDEO A/B TEST ONLY
+			 */
+			if (inGroup('FEATURED_VIDEO_VIEWABILITY_VARIANTS', 'PAGE_PLACEMENT')) {
+				return calculatedHeight > 500 ? 500 : calculatedHeight;
```

Comment:
Why 500? If that is enforced by the test rules, how about moving it to a variable so that it's more obvious?

Reply:
it's one of requirements, I can make a variable of it